src/redux/movieSlice.js

```diff
@@ -76,7 +74,7 @@ const movieSlice = createSlice({
     },
     [fetchMovieImages.fulfilled]: (state, action) => {
       state.imagesStatus = 'resolved';
-      state.movieImages = action.payload.backdrops;
+      state.movieImages = action.payload.backdrops.slice(0,8);
```

Comment:
why `slice(0,8)`?

Reply:
in the mock-up file we have only 8 images